src/javascript/app/base/header.js

```diff
@@ -265,44 +265,61 @@ const Header = (() => {
 
             const riskAssessment = () => (Client.getRiskAssessment() && !Client.isJPClient());
 
+            const hasMissingRequiredField = () => {
+                const required_field = [
+                    'account_opening_reason',
+                    'address_line_1',
+                    'address_city',
+                    'phone',
+                    'tax_residence',
+                    'tax_identification_number',
+                    ...Client.get('residence') === 'gb' ? 'address_postcode' : '',
+                ];
+                const get_settings = State.getResponse('get_settings');
+                return required_field.some(field => get_settings[field] === '');
```

Comment:
To handle possible API inconsistencies, I'd suggest `...some(field => !get_settings[field]);`